src/main/java/com/github/dockerjava/api/model/HostConfig.java

```diff
@@ -27,6 +27,9 @@
 	@JsonProperty("Privileged")
 	private boolean privileged;
 
+	@JsonProperty("ReadonlyRootfs")
+	private boolean readonlyRootfs;
```

Comment:
Well, in this case a default boolean = false will be ok as long as ReadonlyRootfs option is false by default. I think it'll we better if we add default values in such classes corresponding to docker defaults.